spec/support/omniauth_helpers.rb

```diff
@@ -0,0 +1,17 @@
+def set_omniauth(opts = {})
```

Comment:
Unused method argument - `opts`. If it's necessary, use `_` or `_opts` as an argument name to indicate that it won't be used. You can also write as `set_omniauth(*)` if you want the method to accept any arguments but don't care about them.